source/modes/GameEditorModeBase.cpp

```diff
@@ -132,6 +133,12 @@ GameEditorModeBase::~GameEditorModeBase()
     // Delete the potential pending event messages
     for (EventMessage* message : mEventMessages)
         delete message;
+
+    // Clear up any events and chat messages.
+    mRootWindow->getChild("GameChatWindow/GameChatText")->setText("");
```

Comment:
No, the secret is to check what have been done in the conflicting PR and do the same. Here, some code had been moved and you re-added it.